projects/ngx-quill/src/lib/quill-view.component.ts

```diff
@@ -152,9 +152,15 @@ export class QuillViewComponent implements AfterViewInit, OnChanges {
       this.valueSetter(this.quillEditor, this.content)
     }
 
-    // trigger created in a timeout to avoid changed models after checked
-    setTimeout(() => {
-      this.onEditorCreated.emit(this.quillEditor)
-    })
+    // The `requestAnimationFrame` triggers change detection. There's no sense to invoke the `requestAnimationFrame` if anyone is
+    // listening to the `onEditorCreated` event inside the template, for instance `<quill-view (onEditorCreated)="...">`.
+    if (this.onEditorCreated.observers.length > 0) {
```

Comment:
could you just please use `!this.onEditorCreated.observers.length` so it is more unified with the other observers checks?

Thanks :)

Reply:
Updated to `return` early (took from other places).

Reply:
yes thats okay, but just to have a unified style to handle it. Thanks